src/efisecdb.c

```diff
@@ -332,6 +330,15 @@ static list_t infiles;
 static list_t actions;
 static char *outfile = NULL;
 
+static void
+on_exit_cleanup(void)
```

Comment:
I think this function isn't needed - you can just make each cleanup function a valid atexit() and call them that way.

Reply:
Yes, but I generally don't like global vars. I kinda liked that the only use of the global vars are in one single place, well visible where they are declared.

But yeah, I can do multiple atexit calls if you prefer that.

Reply:
That's an interesting way to think about it.  If it helps, the way we think of it is maybe closer to... when the variable is set up, instructions are also given for how to tear it down.  And the atexit/on_exit infrastructure takes care of that for us, so we don't have to keep track of it.